core/io/CMakeLists.txt

```diff
@@ -1,12 +1,24 @@
 add_library(couchbase_io STATIC mcbp_message.cxx mcbp_parser.cxx http_parser.cxx dns_config.cxx)
 set_target_properties(couchbase_io PROPERTIES POSITION_INDEPENDENT_CODE ON)
 add_dependencies(couchbase_io http_parser)
-target_link_libraries(
-  couchbase_io
-  PRIVATE project_options
-          project_warnings
-          snappy
-          http_parser
-          fmt::fmt
-          spdlog::spdlog)
+if(WIN32)
+    target_link_libraries(
+            couchbase_io
+            PRIVATE project_options
+            project_warnings
+            snappy
+            http_parser
+            fmt::fmt
+            spdlog::spdlog
+            iphlpapi)
+else()
+    target_link_libraries(
+            couchbase_io
+            PRIVATE project_options
+            project_warnings
+            snappy
+            http_parser
+            fmt::fmt
+            spdlog::spdlog)
+endif()
```

Comment:
Also looks like you can use multiple `target_link_libraries` calls, and guard only `lphlpapi`.

Reply:
I didn't know it was additive -- cool!  Updates to guard only `iphlpapi` 👍